pages/api/v1/auth/actions/followUser.ts

```
@@ -15,50 +15,87 @@ import messages from '~src/auth/utils/messages';
+	}
 
 	const network = String(req.headers['x-network']);
-	if (!network || !isValidNetwork(network)) return res.status(400).json({ message: 'Missing or invalid network name in request headers' });
+	if (!network || !isValidNetwork(network)) {
+		return res.status(400).json({ message: 'Missing or invalid network name in request headers' });
+	}
 
-	// userId to follow
 	const { userId } = req.body;
-	if (isNaN(Number(userId)) || userId === null || userId === undefined) return res.status(400).json({ message: 'Missing or invalid user id in request body' });
+	if (!userId || isNaN(Number(userId))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)